Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -171,11 +160,47 @@ void UPSWorldSubsystem::OnWorldBeginPlay(UWorld& InWorld)
 void UPSWorldSubsystem::Deinitialize()
 {
 	Super::Deinitialize();
-	PSHUDComponentInternal = nullptr;
-	PSCurrentSpotComponentInternal = nullptr;
-	SaveGameDataInternal = nullptr;
-	ProgressionSettingsDataInternal.Empty();
-	StarDynamicProgressMaterial = nullptr;
+}
+
+// Invoked after a game feature plugin is unloaded
+void UPSWorldSubsystem::OnGameFeatureUnloading(const UGameFeatureData* GameFeatureData, const FString& PluginURL)
+{
+	DestroyStarActors();
+	IGameFeatureStateChangeObserver::OnGameFeatureUnloading(GameFeatureData, PluginURL);
```

Comment:
You don't need call interface method here, interfaces always empty in base class by design
```
IGameFeatureStateChangeObserver::OnGameFeatureUnloading
```

The same in `OnGameFeatureLoading`

Reply:
done as part of: https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/82181f3302a54cac90bd83f4434335acf4bb5eb4